examples/hello_world/client/rust/src/main.rs

```diff
@@ -0,0 +1,107 @@
+//
+// Copyright 2020 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+//! Hello World client that sends 4 concurrent
+
+#![feature(async_closure)]
+
+use hello_world_grpc::proto::{hello_world_client::HelloWorldClient, HelloRequest};
+use log::info;
+use oak_abi::label::Label;
+use prost::Message;
+use structopt::StructOpt;
+use tonic::{
+    metadata::MetadataValue,
+    transport::{Certificate, Channel, ClientTlsConfig},
+    Request,
+};
+#[derive(StructOpt, Clone)]
+#[structopt(about = "Hello World Client")]
+pub struct Opt {
+    #[structopt(
+        long,
+        help = "URI of the Oak application to connect to",
+        default_value = "https://localhost:8080"
+    )]
+    uri: String,
+    #[structopt(
+        long,
+        help = "PEM encoded X.509 TLS root certificate file used by gRPC client"
+    )]
+    root_tls_certificate: String,
+}
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    info!("Initialising and reading args");
+    env_logger::init();
+    let opt = Opt::from_args();
+
+    let uri = opt.uri.parse().expect("Error parsing URI");
+    let root_tls_certificate = tokio::fs::read(&opt.root_tls_certificate)
+        .await
+        .expect("Could not load certificate file");
+
+    info!("Connecting to Oak Application: {:?}", uri);
```

Comment:
Do we need `{:?}` instead of `{}` isn't `uri` a string at this point?

Reply:
I think it's actually a http::uri